static/js/controllers/reports.js

```diff
@@ -194,14 +282,17 @@ var _ = require('underscore'),
             $scope.error = false;
             $scope.errorSyntax = false;
             _updateLiveList(data);
```

Comment:
also, it's not from this PR changes, but l.69 (I can't add comment there because it's not edited :( ), there's `liveList.update(report, false)`, and I think the second arg is ignored. I don't know what it's meant to be.

But to pass the checked status, you could do `liveList.update(report, isChecked)`.